src/components/SwipeViews.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable react/no-string-refs */
+import React from 'react';
+import ReactSwipe from 'react-swipe';
+import { Row, Col, Glyphicon } from 'react-bootstrap';
+
+type SwipeViewsProps = {
+  /** array of different views/screens (anything that can be rendered) */
+  views: Array<any>,
+};
+
+class SwipeViews extends React.Component<void, SwipeViewsProps, void> {
+  constructor(props) {
+    super(props);
+    (this: any).next = this.next.bind(this);
+    (this: any).prev = this.prev.bind(this);
+  }
+
+  static displayName = 'SwipeViews';
+
+  next() {
+    this.refs.reactSwipe.next();
+  }
+
+  prev() {
+    this.refs.reactSwipe.prev();
+  }
+
+  render() {
+    const swipeOptions = {
```

Comment:
Do we want to expose any of the values in `swipeOptions` as properties of the component?

Reply:
Done, all of the options are configurable now.